service/pokedex.go

```diff
@@ -0,0 +1,72 @@
+package service
+
+import (
+    "os"
+    "fmt"
+    "strings"
+    "strconv"
+
+    "pokedex/config"
+    "pokedex/model"
+    "pokedex/model/errors"
+    "pokedex/data"
+)
+
+/* This map will act as an in-RAM Pokedex */
+type Pokedex map[int]model.Pokemon
+
+/* Parses data read from CSV into a Pokedex map so it can be consumed later */
+func (pokedex Pokedex) Init() {
+    path, err := os.Getwd()
+    if err != nil {
+        fmt.Println(err)
```

Comment:
Handle the error, what's the purpose of checking for an error if you are not doing a proper handle of this?